Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2013-06-18
Initial Package Version: 1.14
Upstream Status: Submitted
Origin: Arch
Description: Build with perl-5.18.

http://lists.gnu.org/archive/html/bug-wget/2013-06/msg00046.html

From 2ed1707b5d8be66feb80cccfe8e11e719b52b99a Mon Sep 17 00:00:00 2001
From: Dave Reisner <dreisner@archlinux.org>
Date: Mon, 17 Jun 2013 23:31:46 +0530
Subject: [PATCH] Fix error in texi2pod intriduced with Perl 5.18

---

diff --git a/doc/texi2pod.pl b/doc/texi2pod.pl
index 86c4b18..9db6de1 100755
--- a/doc/texi2pod.pl
+++ b/doc/texi2pod.pl
@@ -291,7 +291,7 @@ while(<$inf>) {
 	if (defined $1) {
             my $thing = $1;
             if ($ic =~ /\@asis/) {
-                $_ = "\n=item $thing\n";
+                $_ = "\n=item C<$thing>\n";
             } else {
                 # Entity escapes prevent munging by the <> processing below.
                 $_ = "\n=item $ic\&LT;$thing\&GT;\n";
-- 
1.8.3.1

